packages/multi-test/src/contracts.rs

```diff
@@ -337,8 +338,13 @@ where
         info: MessageInfo,
         msg: Vec<u8>,
     ) -> AnyResult<Response<C>> {
-        let msg = from_slice(&msg)?;
-        (self.execute_fn)(deps, env, info, msg).map_err(|err| anyhow!(err))
+        let msg: T1 = from_slice(&msg)?;
+        (self.execute_fn)(deps, env, info, msg.clone())
+            .map_err(anyhow::Error::from)
+            .context(format!(
+                "Contract returned an error on execute msg:\n{:?}",
+                msg,
+            ))
     }
 
     fn instantiate(
```

Comment:
We should do the same map error here and for query, right?

I assume that will be a follow-up PR, but just want to double check

Reply:
Ah yeah, I guess I shouldn't close the issue until those are implemented.